Makefile

```diff
@@ -37,14 +37,14 @@ GCFLAGS="all=-N -l"
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.28.0
 
-TEST_MANIFEST_DIR_DEFAULT ?= tests/manifests/default
-TEST_PPND_MANIFEST_DIR ?= tests/manifests/special-cases/ppnd
+TEST_MANIFEST_DIR_PPND ?= tests/manifests/default
+TEST_NOSTRATEGY_MANIFEST_DIR ?= tests/manifests/special-cases/no-strategy
```

Comment:
now we have a `no-strategy` override directory since "pause-and-drain" is the default strategy